packages/generators/generators/lib/plops/controller.js

```diff
@@ -18,7 +18,8 @@ module.exports = plop => {
       ...getDestinationPrompts('controller', plop.getDestBasePath()),
     ],
     actions(answers) {
-      const filePath = getFilePath(answers.destination);
+      const resolveFilePath = (isPlugin,filePath) => isPlugin ? `${filePath}/server` : filePath;
+      const filePath = resolveFilePath(answers.plugin,getFilePath(answers.destination));
```

Comment:
Same as for the CT generator. This should be fixed inside the `getFilePath` method.

Reply:
done